heltour/tournament/templates/tournament/admin/create_teams.html

```diff
@@ -0,0 +1,41 @@
+{% extends "tournament/admin/custom_edit_workflow.html" %}
+
+{% block content %}
+<p>
+Create new teams for season {{ season }}.  This will completely delete all teams and alternates and recreate them
+according to the algorithm in `create_teams.py`.
+</p>
+
+{% if season_started %}
```

Comment:
Please make this impossible to do once the season has started.

Reply:
What is the purpose of the `revision` module if not to undo mistakes like this?  I figured allowing this would make testing easier.  I will now have to delete all the `rounds` in my test db to test this feature in the future.

Reply:
Reversion works great for individual changes. It's also great for viewing a history of changes. It does not provide a good way to revert hundreds of interrelated changes which could potentially also trigger other tasks.

Reply:
Or more generally, catastrophe prevention > catastrophe recovery.

Reply:
If you really want you can allow it when the DEBUG flag is set like [here](https://github.com/cyanfish/heltour/blob/c07d7f698794fd34c699bb43de91fb4181696cfb/heltour/tournament/admin.py#L370).

Reply:
I've already made the change to prevent it when the season is started. Let's roll with this for this PR, and we can discuss how to make it easier to develop in a subsequent one